gfx/drivers/miyoomini/sdl_miyoomini_gfx.c

```diff
@@ -782,6 +787,40 @@ static void *sdl_miyoomini_gfx_init(const video_info_t *video,
 
    sdl_miyoomini_set_cpugovernor(PERFORMANCE);
 
+   const char *fb_device = "/dev/fb0";
+   const char *rgui_menu_stretch_file = "/mnt/SDCARD/.tmp_update/config/RetroArch/.noMenuStretch";
```

Comment:
Might prefer that this was a macro, i.e. `#define RGUI_MENU_STRETCH_PATH "[...]"`

Reply:
all three of the strings there?